packages/cdk8s-cli/jest.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  "collectCoverage": true,
+  "moduleFileExtensions": [
+    "js"
```

Comment:
omit

Reply:
Do you mean centralize, or can we get rid of the `moduleFileExtensions` rule?

Reply:
yes, centralize. I don't want any duplication between packages... I see this as an important benefit of Bazel that we should double down on.